src/PrestaShopBundle/Resources/config/routing/admin/sell/customer/outstanding.yml

```diff
@@ -0,0 +1,16 @@
+admin_outstanding_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Customer/Outstanding:index
+    _legacy_controller: AdminOutstanding
+    _legacy_link: AdminOutstanding
+
+admin_outstanding_search:
+  path: /
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Common:searchGrid
```

Comment:
If you use common search action in this route, then `PrestaShopBundle:Admin/Sell/Customer/Outstanding:search` is not necessary? 🤔

Reply:
Changed this to use custom one, because common one need grid definition extending with special class.

Reply:
All right 👍